CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix typos found across the codebase
```

Comment:
I think it would be better to append that at the bottom of the list, what do you think? New entries usually come there